collection-scripts/gather_priority_and_fairness

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# If the cluster version is < 4.5 skip this test
+VERSION=$(oc get clusterversion version -o jsonpath='{.status.desired.version}')
+COMPARE=$(echo -e "4.5\n${VERSION}" | sort -V | head -n1)
+if [[ ${COMPARE} != "4.5" ]]; then
```

Comment:
I tried the following endpoints on a 4.5 cluster today and they were not available on that version.
I will check if we have them in 4.6 and report back here.

/hold

Reply:
The debug endpoints were added in 1.19 (upstream). OpenShift 4.6 is based on that version. 

The endpoints are defined here https://github.com/openshift/kubernetes/blob/release-4.6/staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller_debug.go#L41

It looks like we need to update the check to skip if the version is < 4.6

Reply:
Updated to >=4.6

Reply:
thanks.

/hold cancel